src/java.base/share/classes/java/text/DigitList.java

```diff
@@ -161,12 +161,12 @@ public final double getDouble() {
             return 0.0;
```

Comment:
The doc for this method is incorrect:
```
* If (count == 0) this throws a NumberFormatException, which
* mimics Long.parseLong().
```

Reply:
I believe that relates to https://bugs.openjdk.org/browse/JDK-8170389

Which I can address in a separate PR

Reply:
Good catch! Also, I think the comment should compare it with `Double.parseDouble()`